packages/shorebird_cli/lib/src/executables/gradlew.dart

```diff
@@ -113,3 +117,16 @@ class Gradlew {
     return productFlavors;
   }
 }
+
+extension on String {
+  bool get isAcronymPrefixed {
+    if (length >= 2) {
+      return this[0].isUpperCase() && this[1].isUpperCase();
+    }
+    return false;
+  }
+
+  bool isUpperCase() {
+    return this == toUpperCase();
+  }
```

Comment:
This seems general enough to go in `shorebird_cli/lib/src/extensions/string.dart`